src/EventHub/EventHub/help/New-AzEventHubNamespace.md

```diff
@@ -1,37 +1,37 @@
 ﻿---
 external help file: Microsoft.Azure.PowerShell.Cmdlets.EventHub.dll-Help.xml
 Module Name: Az.EventHub
-online version: https://docs.microsoft.com/en-us/powershell/module/az.eventhub/new-azeventhubnamespace
+online version:
 schema: 2.0.0
 ---
 
 # New-AzEventHubNamespace
 
 ## SYNOPSIS
-Creates an Event Hubs namespace.
+{{ Fill in the Synopsis }}
```

Comment:
Please regenerate the help document. `online version` and `SYNOPSIS` should not be empty.

Reply:
updated